SuperNETorg/BarterDEX
# Feature/58 auto select buy or sell tab

This PR implements **feature request https://github.com/SuperNETorg/BarterDEX/issues/58.**

**Demo:** https://youtu.be/R7mn7SwMlEI

How to test:

1. Open BarterDEX
2. Log in using your passphrase
3. Add some coins
4. Click "Exchange" next to one of the coins in the list
5. Check if you see an order book with sellers and buyers
6. Click an order in the "Buyers" list: The "Sell $COIN" tab will be auto selected
7. Click an order in the "Sellers" list: The "Buy $COIN" tab will be auto selected

____

Please let me know if this way of contributing is as you wish. If not, please create a [`CONTRIBUTE.md`](https://github.com/SuperNETorg/BarterDEX/community) file in the root of this repository that explains the best way to contribute :-)
